recipe/scripts/deactivate.sh

```diff
@@ -4,7 +4,7 @@
 unset PROJ_LIB
 unset PROJ_NETWORK
 
-if [ -n "$_CONDA_SET_PROJ_LIB" ]; then
-    export PROJ_LIB=$_CONDA_SET_PROJ_LIB
+if [ -n "${_CONDA_SET_PROJ_LIB:-}" ]; then
+    export PROJ_LIB="${_CONDA_SET_PROJ_LIB}"
     unset _CONDA_SET_PROJ_LIB
 fi
```

Comment:
@TormodLandet, I'm a little confused by why the previous version didn't work for you.  I think the problem was in the if, which I have not changed since last commit and is identical to what you requested.  I changed the `export` line to match your example.  See if this works as expected.

Reply:
I may have clicked on the wrong link, anyway the code in the PR now works